docs/source/tutorials/EUREF2019/answers.rst

```diff
@@ -13,7 +13,7 @@ Projections.1: UTM
     3.      +proj=utm +zone=35 +ellps=intl
 
 
-Projections2.: LCC
+Projections.2: LCC
```

Comment:
Shouldn't the dot just be removed all together?

Reply:
I don't know... maybe? The format with the dot is used above in `Projection.1` and below in `Projection.3`.
This change is only meant to align the formatting.

Reply:
I think they were supposed to reflect the naming of the gie files. Så `projection1` etc would be correct

Reply:
I see. Makes sense. I removed the dots.